core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -516,6 +521,18 @@ private void syncRemoteChangeEventToLocal(
         remoteChangeEvent);
   }
 
+  private void emitError(BsonValue docId, String msg) {
```

Comment:
[minor] Similar to emitting events, we should use the dispatcher for this so that we can continue on with our code. Right now we can block forever depending on what the user's callback does.